app/classes/observation_report/mycoflora.rb

```diff
@@ -1,12 +1,16 @@
 module ObservationReport
   # Format for export to Mycoflora.
   class Mycoflora < ObservationReport::CSV
+    MYCOFLORA_PROJECT_NAME = "North American Mycoflora Project".freeze
```

Comment:
Is there a better place to define this constant so that it can easily be used outside this file, e.g., tests, ObserverController::make_labels, etc.?